src/client/proposedApiTypes.ts

```diff
@@ -1,4 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export interface ProposedExtensionAPI {}
+export interface ProposedExtensionAPI {
```

Comment:
```suggestion
export interface ProposedExtensionAPI extends ProposedCreateEnvironmentAPI {
```

Contents of `ProposedCreateEnvironmentAPI` should be defined here, we can either extend it or add all the new APIs.